example/symbol_exec/single_instr.py

```diff
@@ -1,36 +1,38 @@
 # Minimalist Symbol Exec example
-from miasm2.core.bin_stream import bin_stream_str
 from miasm2.ir.symbexec import SymbolicExecutionEngine
 from miasm2.analysis.machine import Machine
 from miasm2.core.asmblock import AsmSymbolPool
+from miasm2.core import parse_asm
 
 START_ADDR = 0
 machine = Machine("x86_32")
 
 symbol_pool = AsmSymbolPool()
 
 
-# Assemble and disassemble a MOV
-## Ensure that attributes 'offset' and 'l' are set
-line = machine.mn.fromstring("MOV EAX, EBX", symbol_pool, 32)
```

Comment:
AFAIK, the purpose of this example was the use of `fromstring`. `parse_txt` API is already illustrated in scripts in the same dir